getting-started/ts-example/README.md

```diff
@@ -0,0 +1,353 @@
+
+#### Install the required OpenTelemetry libraries
+
+([link to JavaScipt version](../README.md#install-the-required-opentelemetry-libraries))
+
+To create traces on NodeJS, you will need `@opentelemetry/node`, `@opentelemetry/core`, and any plugins required by your application such as gRPC, or HTTP. If you are using the example application, you will need to install `@opentelemetry/plugin-http`.
+
+```sh
+$ npm install \
+  @opentelemetry/core \
+  @opentelemetry/node \
+  @opentelemetry/plugin-http \
+  @opentelemetry/types
+```
+
+#### Initialize a global tracer
+
+([link to JavaScipt version](../README.md#initialize-a-global-tracer))
+
+All tracing initialization should happen before your application’s code runs. The easiest way to do this is to initialize tracing in a separate file that is required using node’s `-r` option before application code runs.
+
+Create a file named `tracing.ts` and add the following code:
+
+```typescript
+import * as opentelemetry from "@opentelemetry/core";
+import { NodeTracer } from "@opentelemetry/node";
+
+const tracer: NodeTracer = new NodeTracer({
+  logLevel: opentelemetry.LogLevel.ERROR
+});
+
+opentelemetry.initGlobalTracer(tracer);
+```
+
+If you run your application now with `ts-node -r ./tracing.ts app.ts`, your application will create and propagate traces over HTTP. If an already instrumented service that supports [Trace Context](https://www.w3.org/TR/trace-context/) headers calls your application using HTTP, and you call another application using HTTP, the Trace Context headers will be correctly propagated.
+
+If you wish to see a completed trace, however, there is one more step. You must register an exporter to send traces to a tracing backend.
+
+#### Initialize and Register a Trace Exporter
+
+([link to JavaScipt version](../README.md#initialize-and-register-a-trace-exporter))
+
+This guide uses the Zipkin tracing backend, but if you are using another backend like [Jaeger](https://www.jaegertracing.io), this is where you would make your change.
+
+To export traces, we will need a few more dependencies. Install them with the following command:
+
+```sh
+$ npm install \
+  @opentelemetry/tracing \
+  @opentelemetry/exporter-zipkin
+
+$ # for jaeger you would run this command:
+$ # npm install @opentelemetry/exporter-jaeger
+```
+
+After these dependencies are installed, we will need to initialize and register them. Modify `tracing.ts` so that it matches the following code snippet, replacing the service name `"getting-started"` with your own service name if you wish.
+
+```typescript
+import * as opentelemetry from "@opentelemetry/core";
+import { NodeTracer } from "@opentelemetry/node";
+
+import { SimpleSpanProcessor } from "@opentelemetry/tracing";
+import { ZipkinExporter } from "@opentelemetry/exporter-zipkin";
+
+const tracer: NodeTracer = new NodeTracer({
+  logLevel: opentelemetry.LogLevel.ERROR
+});
+
+opentelemetry.initGlobalTracer(tracer);
+
+tracer.addSpanProcessor(
```

Comment:
I would add:

```ts
// import { JaegerExporter } from "@opentelemetry/exporter-jaeger";
// tracer.addSpanProcessor(
//  new SimpleSpanProcessor(
//    new JaegerExporter({
//      serviceName: "getting-started"
      // If you are running your tracing backend on another host,
      // you can point to it using the `url` parameter of the
      // exporter config.
//    })
//  )
// );
```

Reply:
Updated, please take a look again :)

Reply:
Instead of the whole blob of text, I added four lines to where the code would be different (see above and below this line in the file)